### PR TITLE
Add a slide-in animation for the 'next' button

### PR DESCRIPTION
**Pull request by DanielHeath**

### CODE REVIEW

**Comment by danni** [comment]
Can we maybe use one of the SASS CSS animation mixins?

**Comment by DanielHeath** [comment]
I generally avoid abstracting things if there's only one instance of it. Are you happy to revisit that if / when we add a second animation?

**Comment by danni** [comment]
I mean one of the existing frameworks. Rather than reinventing a wheel. Especially since other animations, i.e. page transitions will be a thing we want in the future.

**Comment by danni** [comment]
Instead of adding Saffron in our repo why not add a `bower.json` ?

**Comment by danni** [comment]
And then have `bower_components` ignored by Git the same way `node_modules` is.

**Comment by DanielHeath** [comment]
@danni ++?

File: .travis.yml [hunk 1]
```diff
@@ -10,6 +10,7 @@ before_install:
 install:
   - npm config set cache ./node_modules_cache
   - npm install
+  - node_modules/.bin/bower install
```
Comment (danni) [inline .travis.yml]: `npm bin` ?

File: Dockerfile [hunk 2]
```diff
@@ -44,6 +44,8 @@ COPY package.json /app/
 RUN npm install && \
     npm cache clean
 
+RUN node_modules/.bin/bower install
```
Comment (danni) [inline Dockerfile]: Combine with the line above and clean the cache afterwards. Also `npm bin`

File: bower.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  "private": true,
+  "ignore": [
+    "**/.*",
+    "node_modules",
+    "bower_components",
+    "test",
+    "tests"
+  ],
+  "dependencies": {
+    "saffron": "~0.2.3"
```
Comment (danni) [inline bower.json]: Remove `~`. Add this to your typecheck.

File: package.json [hunk 4]
```diff
@@ -11,6 +11,7 @@
     "babel-core": "5.8.25",
     "babel-loader": "5.3.2",
     "babel-runtime": "5.8.25",
+    "bower": "1.6.5",
```
Comment (danni) [inline package.json]: Probably just a dev dep?

File: .gitignore [hunk 5]
```diff
@@ -9,3 +9,4 @@
 /public/static/*.js
 /public/static/*.js.map
 /src/server/webpack-stats.json
+bower_components
```
Comment (danni) [inline .gitignore]: `/`

Comment (danni) [inline .gitignore]: Also sorting.

**Comment by danni** [comment]
Otherwise ++

**Pull request merged**